.circleci/bin/test-airflow-image.py

```diff
@@ -106,6 +106,90 @@ def test_astronomer_airflow_check_version(webserver):
         "astronomer-airflow-version-check module must be greater than 1.0.0"
 
 
+def test_airflow_connections(scheduler):
+    """Test Connections can be added and deleted"""
+    test_conn_uri = "postgresql://postgres_user:postgres_test@1.1.1.1:5432"
+    test_conn_id = "test"
+
+    # Assert Connection can be added
+    assert f"Successfully added `conn_id`={test_conn_id} : {test_conn_uri}" in scheduler.check_output(
+        f'airflow connections -a --conn_uri "{test_conn_uri}" --conn_id {test_conn_id}')
+
+    # Assert Connection can be removed
+    assert f"Successfully deleted `conn_id`={test_conn_id}" in scheduler.check_output(
+        f'airflow connections -d --conn_id {test_conn_id}')
+
+
+def test_airflow_variables(scheduler):
+    """Test Variables can be added, retrieved and deleted"""
+    # Assert Variables can be added
+    assert "" in scheduler.check_output("airflow variables --set test_key test_value")
+
+    # Assert Variables can be retrieved
+    assert "test_value" in scheduler.check_output("airflow variables --get test_key")
+
+    # Assert Variables can be deleted
+    assert "" in scheduler.check_output("airflow variables --delete test_key")
+
+
+def create_example_dag(scheduler):
+    """
+    Create Example DAG and add it to Scheduler POD
+    """
+    example_dag = """
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from airflow.utils.timezone import datetime
+
+dag = DAG(
+    dag_id="example_dag",
+    default_args={"start_date": datetime(2020, 5, 1), "owner": "airflow"},
+    schedule_interval=None,
+)
+
+bash_task = BashOperator(
+    task_id="bash_task",
+    bash_command="echo Test",
+    dag=dag,
+)
+        """
+
+    # Add Example DAG
+    print(scheduler.check_output(f"/entrypoint mkdir -p dags && ls -ltr"))
+    scheduler.check_output(f"/entrypoint touch dags/example_dag.py ")
+    scheduler.check_output(f"/entrypoint chmod 777 dags/example_dag.py ")
+    scheduler.check_output(f"echo '{example_dag}' > dags/example_dag.py ")
```

Comment:
```suggestion
    scheduler.check_output(f"echo '{shlex.quote(example_dag)}' > dags/example_dag.py ")
```

Reply:
Changed the way I did this, removed the hacks (or replaced with other hacks)